contrib/kubespray/doc/quick-start.md

```diff
@@ -0,0 +1,48 @@
+### Quick Start
+
```

Comment:
before the major content, you might want to add a brief introduction what's needed to be updated and a short introduction of each files.

Reply:
Done